OPHD/States/MapViewStateTurn.cpp

```diff
@@ -517,7 +517,11 @@ void MapViewState::nextTurn()
 	updateStructuresAvailability();
 	updateRoads();
 
+	// Overlay Updates
 	checkCommRangeOverlay();
+	btnToggleConnectednessClicked();
+	btnToggleCommRangeOverlayClicked();
```

Comment:
This seems a bit hacky. Might be a sign the button handler code should be refactored so it can be called directly.

Reply:
It absolutely is. It's mostly just avoiding pulling code out of the buttons because... well... it effectively works.

That said, the code between the two buttons is very very similar. It will change a bit going forward as new overlay buttons are added (there's really only room for one more so not by much). It could use some attention.